openwisp_controller/notifications/tests/test_notifications.py

```diff
@@ -0,0 +1,64 @@
+from django.apps.registry import apps
+from django.test import TestCase
+from openwisp_notifications.types import unregister_notification_type
+from swapper import load_model
+
+from openwisp_controller.config.tests.utils import CreateConfigMixin
+from openwisp_users.tests.utils import TestOrganizationMixin
+
+Config = load_model('config', 'Config')
+Notification = load_model('openwisp_notifications', 'Notification')
+
+notification_qs = Notification.objects.all()
+
+
+class TestConfigNotifications(CreateConfigMixin, TestOrganizationMixin, TestCase):
```

Comment:
this can be moved as follows:

file: `openwisp_controller.config.tests.test_notifications`
class name: `TestNotifications`